example/profile.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require_once __DIR__ . '/../src/boot.php';
+require_once './vendor/autoload.php';
+
+$config = [
+    
+    'sdkId' => 'add your SDK ID here',
+    'pemFile' => __DIR__ . '/keys/your-key-name.pem',
+];
+
+$client = new Yoti\YotiClient($config['sdkId'], $config['pemFile']);
+$profile = $client->getActivityDetails($output['token']);
```

Comment:
`$output` will be undefined, you have to read it from the URL like this: `parse_str($_SERVER["QUERY_STRING"], $output);`. I would even change `$output` to something like `$parsedQueryString`